tests/src/test/java/alluxio/client/fs/LoadMetadataIntegrationTest.java

```diff
@@ -219,15 +219,9 @@ public void loadRecursive() throws Exception {
         fileWriter.close();
       }
     }
-    long startMs = CommonUtils.getCurrentMs();
     List<URIStatus> list = mFileSystem.listStatus(new AlluxioURI("/mnt"), options);
-    long durationMs = CommonUtils.getCurrentMs() - startMs;
     // 25 files, 25 level 2 dirs, 5 level 1 dirs, 1 file and 1 dir created in before
     Assert.assertEquals(25 * 2 + 5 + 2, list.size());
-
-    // Should load metadata once, in one recursive call
```

Comment:
Is this test useful anymore, if it is not validating how the load is happening?

Reply:
I think validating the listStatus size is better than validating the runtime? Runtime can vary for a lot of reasons during tests- especially in jenkins if a system is overloaded.

We might be able to come up with some better assertions, but I think the test is still validating that we load metadata since all of the file creations above are through the UFS, and then the listStatus call is executed through Alluxio, so we verify that at least all of the files expected are loaded in Alluxio under the `/mnt` path.